laser_resender/src/laser_resender_node.cpp

```diff
@@ -35,6 +36,7 @@ class LaserResender : public rclcpp_lifecycle::LifecycleNode
   LaserResender()
   : rclcpp_lifecycle::LifecycleNode("laser_resender")
   {
+    declare_parameter("node_name");
```

Comment:
Following the Arne suggestions in https://github.com/micro-ROS/system_modes/issues/45#issuecomment-700538321 I have defined a fake parameter.

> Currently, there has to be a parameter, because otherwise the mode is invisible from the outside and can't be inferred in any way.
> 
> You may want to introduce a simple _fake_ parameter, that is a bool for example or - to make it a bit easier to understand - a string parameter that holds the name of the mode. We stumbled upon this question before.

Reply:
ok